compiler/damlc/daml-stdlib-src/DA/Numeric.daml

```diff
@@ -11,30 +11,60 @@ import GHC.Types (primitive)
 -- different scales, unlike `(*)` which forces all numeric scales
 -- to be the same. Raises an error on overflow, rounds to chosen
 -- scale otherwise.
+#ifdef DAML_NAT_TYPE_ERASURE
+mul : forall n1 n2 n3. Numeric n3 -> Numeric n1 -> Numeric n2 -> Numeric n3
```

Comment:
I'd rather keep the output tyvar first (`forall n3 n1 n2` here) to match the legacy versions. I'd argue it's less frequent to do type applications for the input tyvars since the corresponding subexpressions likely have a type of their own, unlike the dummy `0.0` argument for the output type

Reply:
Good point - it'll keep some of the test cases more consistent, too.